Gruntfile.js

```diff
@@ -333,18 +328,18 @@ module.exports = function(grunt) {
   grunt.registerTask('update', ['bowercopy', 'sed']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('devserver',
-      ['proxymessage', 'express:dev', 'watch']);
+      ['proxymessage', 'express:dev', 'browserify:server', 'watch']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('sdevserver',
-      ['proxymessage', 'express:sdev', 'watch']);
+      ['proxymessage', 'express:sdev', 'browserify:server', 'watch']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('testserver',
-      ['proxymessage', 'express:localtest', 'watch']);
+      ['proxymessage', 'express:localtest', 'browserify:server', 'watch']);
```

Comment:
How is watchify hooked up to the `devserver`, `sdevserver`, and `testserver` tasks?